chevah/compat/tests/normal/test_capabilities.py

```diff
@@ -149,16 +164,17 @@ def test_getAvailablePrivileges(self):
         result = self.capabilities._getAvailablePrivileges()
 
         self.assertIsNotEmpty(result)
-        privilege = self.capabilities._getPrivilegeID(
-            win32security.SE_IMPERSONATE_NAME)
-        self.assertContains((privilege, 3), result)
-
-        privilege = self.capabilities._getPrivilegeID(
-            win32security.SE_SECURITY_NAME)
-        self.assertContains((privilege, 0), result)
 
         if self.runningAsAdministrator():
             privilege = self.capabilities._getPrivilegeID(
+                win32security.SE_SECURITY_NAME)
```

Comment:
I would say that this test should be split into test_getAvailablePrivileges_norma_user and test_getAvailablePrivileges_administrator